states/menu.py

```diff
@@ -5,22 +5,29 @@
 
 
 class MenuOptions(StateOptions):
+    """Variables for Menu options"""
+
     BATTLE = 1
     TEAM = 2
     EXIT = 3
 
 
 class Menu(State):
+    """Displays options for Menu state change"""
```

Comment:
It does a bit more than just display the options, it holds the state of the current menu game state and also lists out options from the player, then reads it from them.

Reply:
Changed to `Displays, holds, and reads options for Menu state change`

Reply:
Minor: make this more consistent with state/battle.py's docstring
`"""Game state for player battling"""`
so maybe go with
`"""Game state for main menu"""`
The Displays, holds, and reads options for state change is a good description, so maybe move that to the State class in states/state.py since essentially all game states will be exhibiting this behavior.

Reply:
Changed the description for `State` in `states/state,py` to `Base class for game state. Displays, holds, and reads options for Menu state change` and the description for `Menu` in `state/menu.py` to `Game state for main menu`